src/dll/edbgEcmdDll.C

```diff
@@ -1675,12 +1681,209 @@ uint32_t dllQueryHostMemInfoRanges( const std::vector<ecmdChipTarget> & i_target
 }
 #endif // ECMD_REMOVE_MEMORY_FUNCTIONS
 
+#ifdef EDBG_ISTEP_CTRL_FUNCTIONS
+
+enum IplMode_t
+{
+    IPL_MODE_NORMAL = 0,
+    IPL_MODE_ISTEP = 1,
+    IPL_MODE_UNKNOWN = 0xff,
+};
+
+static edbgIPLTable g_edbgIPLTable;
+/**
+  * @brief Set the IPL mode
+  *
+  * @param enum ipl_mode i_mode - IPL mode (Interactive or Continous IPL mode)
+  *
+  * @return Upon success, ECMD_SUCCESS will be returned.  A reason code will
+  *         be returned if the execution fails.
+  */
+uint32_t setIplMode(enum IplMode_t i_mode)
```

Comment:
Lib ipl also supports two IPL modes now Default and interactive. I thin k we should call ipl_set_mode() function also as part of this . That way libipl knows IPL is in  istep mode.